### ISSUE TITLE
Dashboard for full dataset

### ISSUE THREAD

**Issue by carterrees** [MAIF/shapash#150]
**Description of Problem**
I'd like to used my trained model to predict on my entire data set and view that in the dashboard. But no matter what I have tried I can only visualize the test data set. Using the titanic code below I traded out Xtest for Xtrain to see if  I could at least visualize the training set for the titanic example. Feeding the Xtrain data set into the prediction call:

Xtrain, Xtest, ytrain, ytest = train_test_split(X_df, y_df, train_size=0.75, random_state=1)

classifier = sklearn.ensemble.RandomForestClassifier(n_estimators=200).fit(Xtrain, ytrain)

y_pred = pd.DataFrame(classifier.predict(Xtrain), columns=['pred'], index=Xtrain.index) # Predictions

# Shape of y_pred is right at (668,1) and has the correct index numbers.
y_pred.shape

# Then I compile: 
xpl = SmartExplainer() # Optional parameter, dict specifies label for features name
xpl.compile(
    x=Xtrain,
    model=classifier,
    preprocessing=encoder, # Optional: compile step can use inverse_transform method
    y_pred=y_pred # Optional
)
# xpl has the right shape and index numbers as well.
xpl.x_pred.shape
xpl.y_pred.shape

# Now when I run the app I still get a visual of only the test data set. Not sure what to do at this point.
app = xpl.run_app(title_story='Titanic')

**Comment by carterrees** [comment]
I'm closing this issue. My port was already in use by another app. app.kill() didn't seem to fully kill it. After restarting my notebook after killing my local server, everything is working.
